histories/elle/cas-fail.edn

```diff
@@ -0,0 +1,13 @@
+; Two processes read initial value nil
+{:process 1, :type :invoke, :f :read, :value nil}
+{:process 1, :type :ok,     :f :read, :value nil}
```

Comment:
Good and bad samples for CAS register are already exist, see `histories/knossos/cas-register/`.